addons/point_of_sale/static/src/js/screens.js

```diff
@@ -963,6 +964,42 @@ var ProductScreenWidget = ScreenWidget.extend({
         if (_.size(this.action_buttons)) {
             this.$('.control-buttons').removeClass('oe_hidden');
         }
+
+        this._onKeyDown = function (event) {
```

Comment:
because we need bind these event on document instead of widget. Main reason behind this is when we change order from switcher our widget loose focus and not receiving events.
![download 1](https://cloud.githubusercontent.com/assets/24876123/26448177/a175e56a-4169-11e7-8c67-8519b3b36efe.png)